src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (result[value] !== undefined) {
```

Comment:
A small suggestion to make it safer (just FYI):
```suggestion
    if (result.hasOwnProperty(value)) {
```

Reply:
Task was approve, but in app has such view. 
![изображение](https://user-images.githubusercontent.com/109860432/222646889-7ff3b979-91d2-4845-8ef3-93087bcd194e.png)

Reply:
Not resolved. If you have any questions - feel free to ask them on `fe_chat`.